src/http_crawler/__init__.py

```diff
@@ -19,7 +19,10 @@ def crawl(base_url, follow_external_links=True):
 
     while todo:
         url = todo.pop()
-        rsp = session.get(url)
+        rsp = session.get(url, allow_redirects=follow_redirects)
+
+        if rsp.status_code // 100 == 3:
+            continue
```

Comment:
Hi @rkrp.  I'm so sorry it's taken me so long to reply to you.

What about `auto_follow_redirects`?

And yes, we should also scrape the body of the 3xx response.